src/app/containers/MostRead/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from 'react';
+import { string } from 'prop-types';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Canonical from './Canonical';
+
+const getLocalMostReadEndpoint = (service, variant) => {
+  const localhostURL = 'http://localhost:7080';
+  const localServiceURL = `${localhostURL}/${service}`;
+
+  return variant
+    ? `${localServiceURL}/most_read/${variant}.json`
+    : `${localServiceURL}/most_read.json`;
+};
+
+const MostReadContainer = ({ endpoint }) => {
+  const { variant } = useContext(RequestContext);
+  const { service } = useContext(ServiceContext);
+
+  return (
+    <Canonical
+      endpoint={endpoint || getLocalMostReadEndpoint(service, variant)}
```

Comment:
In Storybook & in unit tests, we're passing in `endpoint` and ignoring the `getLocalMostReadEndpoint` code. Do we need any of this logic if it's not being used?

Reply:
my initial intention for the mostRead/index.jsx file as to determine whether we should render the amp or canonical version and to dynamically get the url depending on the service. 
so I've updated it now to remove the `endpoint` prop in the index file, and pointed the storybook file to render the canonical directly.

Reply:
Thanks, @thekp!